### ISSUE TITLE
Drop Bower support

### ISSUE THREAD

**Issue by limonte** [sweetalert2/sweetalert2#1375]
Bower has been deprecated for almost 2 years https://github.com/bower/bower/commit/0bd318de53daf9dc3b52bfd7bc753d4d2c2c9948

I guess it's the right time to stop supporting the legacy package manager. People will still be able to install `7.x` or to install the latest version and to use SweetAlert2 as ES module:

```html
<script type="module">
  import Swal from 'bower_components/sweetalert2/src/sweetalert2.js'
  Swal.fire('hi')
</script>
```

**Comment by zenflow** [comment]
@limonte note that script type="module" won't be supported in IE11 (https://caniuse.com/#feat=es6-module)

so another option for those user's using Bower and needing to support IE11: follow Bower's own guide to migrate to Yarn https://bower.io/blog/2017/how-to-migrate-away-from-bower/ .. I would make that the recommended solution for when developers can afford to make the migration

**Comment by limonte** [comment]
Agree, will add that to the release notes of the new major, thanks!

**Comment by limonte** [comment]
:tada: This has been shipped in version 8.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/sweetalert2)
- [GitHub release](https://github.com/sweetalert2/sweetalert2/releases/tag/v8.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
